### ISSUE TITLE
Typo: Amp "Turbulence" is written as "Turbulnce" (see e.g. the Giant Hall front panel)

### ISSUE THREAD

**Issue by xparq** [lunakid/MuLab#48]

**Comment by mutools** [comment]
It's not a typo. Parameter labels are auto shortened if there is not enough place for the full label.

**Comment by xparq** [comment]
_(Hey, nice to see you here! :) Thanks for commenting. I saw your activity earlier, I was just too busy with other things.)_

OK, I did actually think that something like that might as well be the case, just deemed it unlikely eventually. :) Closing this issue then.

BTW, how does that auto-shortening work? Is it worth the trouble, instead of just, say, replacing the removed ending with a dot (making sure at least 2 letters are cut), so even if one might consider "Turbulnce" perhaps somewhat better than "Turbulen.", does the algorithm tend to win over in general? It might be quite impressive, if it does. (E.g. I found this interesting approach, but it uses plenty of crowdsourced data + a net API: https://makemis.com/2018/01/10/abbreviation/)
